[DB] Fix issue #39. Response error

keran-w opened the pull request:
## Decription
The PR fixes the issue that the DB request returns null, as described in https://github.com/Untellable/210-Team4-TDD/issues/39.

## Fixes
- Change add follower/following save logic: save the account info directly. However, this leads to a new problem of querying much slower
- Set a time interval to check whether the db returns the data repetitively. It's a hacky way, but this works.

Note: sometimes, the DB requests still return null.

alexluo1998 reviewed (changes requested):
seems like there are some lint error and the unit tests need to be updated. I can help with the unit tests tonight if needed

keran-w commented:
> seems like there are some lint error and the unit tests need to be updated. I can help with the unit tests tonight if needed

I changed the unit tests and also fixed the lint error so that it passed all checks.
